music_assistant/providers/resonate/player.py

```diff
@@ -328,7 +417,8 @@ async def _on_queue_update(self, event: MassEvent) -> None:
                     current_item.media_item
                 )
                 if image_data is not None:
-                    await asyncio.to_thread(self._update_media_art, image_data)
+                    image = await asyncio.to_thread(Image.open, BytesIO(image_data))
```

Comment:
I don't think it is doing I/O if it opens an image that is stored in memory? In that case it wouldn't need a thread.